server/routes/api/results.js

```diff
@@ -0,0 +1,28 @@
+//results.js
+
+const express = require("express");
+const router = express.Router();
+
+//External Modules
+import { getVotes } from "../utils/getVotes";
+
+router.get("/results", (req, res) => {
+    if (req.body === null) {
+        res.send({ status: 400, error: "No poll id provided" });
+    } else {
+        try {
+            getVotes(req.body.pollId).then(results => {
```

Comment:
When sending get request, I can attach data as params not in the body, so please access request data as 'req.params.pollId'

Reply:
Hey Olga, it should work if you send it as params as Express is using middleware called bodyparser to take the urlencoded data (the params) and add them to a body object in the request.  At least that's what is supposed to happen.  Did you try sending a request to it?  did it fail?

Reply:
That's really working!